src/client/fontengine.cpp

```diff
@@ -271,18 +271,8 @@ gui::IGUIFont *FontEngine::initFont(const FontSpec &spec)
 	assert(spec.size != FONT_SIZE_UNSPECIFIED);
 
 	std::string setting_prefix = "";
-
-	switch (spec.mode) {
-		case FM_Fallback:
-			setting_prefix = "fallback_";
-			break;
-		case FM_Mono:
-		case FM_SimpleMono:
-			setting_prefix = "mono_";
-			break;
-		default:
-			break;
-	}
+	if (spec.mode == FM_Mono)
```

Comment:
What happened to "SimpleMono"?

Reply:
That never reaches this method, `initSimpleFont` is called instead.